classes/class.tapestry-h5p.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require_once dirname(__FILE__) . "/../interfaces/interface.tapestry-h5p.php";
+
+class TapestryH5P implements ITapestryH5P
+{
+  private $postId;
+
+  public function __construct($postId = null)
+  {
+    $this->postId = $postId;
+  }
+
+  public function get()
```

Comment:
All public PHP functions must have a comment block including description, parameters, and return